### PR TITLE
contributing.md create

### LINKED ISSUE #62: Add a contribution outline doc

### PR DESCRIPTION
**Pull request by LuizHAP**
Fix #62

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
```
Comment (jaepass) [inline CONTRIBUTING.md:8]: These steps do not apply to the project entirely. Would be great to edit to include steps like forking, cloning of the project and making the PR.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (jaepass) [inline CONTRIBUTING.md:76]: Include our email hello@commercejs.com

**Review by jaepass — CHANGES_REQUESTED** [review]
Thanks Luiz for the PR. Would be great to also specify Commerce.js where applicable in the contributing guide.

**Comment by LuizHAP** [comment]
fixed @jaepass

**Comment by jaepass** [comment]
Hi Luiz, apologies but I can't accept the changes as is for this project. Feel free to have a read of this [contributing guide](https://github.com/chec/commerce.js/blob/master/CONTRIBUTING.md) to draw from. Good suggestion might also be to spin up this project and test out how to contribute.

**Comment by LuizHAP** [comment]
@jaepass 
So I can go up the informed link and add the information that is in it, is that it?

**Comment by jaepass** [comment]
Hi @LuizHAP not just extract the exact info but use as a reference to apply to this project specifically. I'll close this pr for the time being please feel free to open another one when it is ready thank you.

**Pull request closed without merging**